tests/linen/linen_test.py

```diff
@@ -1421,12 +1450,10 @@ def body_fun(carry, _):
     # 2nd iteration
     grads, new_ah, new_sf = outer_fn(3., new_ah, new_sf)
     np.testing.assert_allclose(new_ah, [3., 0., 2.])
-    np.testing.assert_allclose(new_sf, [2. / 448])
+    np.testing.assert_allclose(new_sf, [2. / e4m3_max])
     # 3rd iteration
     grads, new_ah, new_sf = outer_fn(4., new_ah, new_sf)
     np.testing.assert_allclose(new_ah, [4., 2., 3.])
-    np.testing.assert_allclose(new_sf, [3. / 448])
+    np.testing.assert_allclose(new_sf, [3. / e4m3_max])
 
 
-if __name__ == '__main__':
```

Comment:
uh, don't delete this.  many test runners won't run tests without it.

Reply:
Sorry, it was accidentally removed. Didn't mean to. I just restored it.